components/Layout/layout.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+import { Header } from '@/components';
+
+export default function Layout({
+  children,
+}: {
+  children: ReactNode
```

Comment:
What is a ReactNode?

Reply:
https://letmegooglethat.com/?q=What+is+a+ReactNode%3F